### ISSUE TITLE
stylish-haskell is run in `/`

### ISSUE THREAD

**Issue by cblp** [atom-haskell/ide-haskell#151]
I found that *stylish-haskell* don't read my project's `.stylish-haskell.yaml`.

I wrapped `stylish-haskell` in a simple script and printed `pwd` into a file, pwd was `/`.

I think problem is in this code (`lib/binutils/prettify.coffee`):
```coffee
    try
      workDir = dirname(editor.getPath())
      if not statSync(workDir).isDirectory()
        workDir = '.'
    catch
      workDir = '.'
```

Maybe some exception is thrown which shouldn't be caught this way.

Atom is not started from '/', it is started from the project's directory, like `~/dev/myproject/`.

**Comment by cblp** [comment]
I know neither JavaScript nor CoffeScript, but `catch` without specifying the type of exceptions looks strange.

**Comment by cblp** [comment]
Almost fixed with #153. But I still don't know how '.' becomes '/'. Maybe you shouldn't use '.' at all.
